district_data_import.py

```diff
@@ -12,7 +12,8 @@
 FILENAME_NEWDONOR = '03-new-donors.csv'
 FILENAME_DONOR_UPDATES = '04-donor-updates.csv'
 FILENAME_DONOR_UPDATE_MESSAGES = '05-donor-manual-updates.txt'
-
+FILENAME_INFORMAL_SAL_UPDATES = 'REF-informal-sal-updates.csv'
+FILENAME_INFORMAL_SAL_PERSONALIZED = 'REF-informal-sal-personalized.csv'
```

Comment:
Are these informal sal files one-time things or permanent? Let's keep one-time things out of the master branch.

Reply:
This is exactly why I had this in a separate branch and wasn't planning to merge it, but it got too confusing so I put everything in one branch for ease of review. Anyway, at this point I can take this out given that we past the point where the separate files would have been useful.